http-ballerina/auth_desugar.bal

```diff
@@ -131,8 +124,8 @@ isolated function getResourceAuthConfig(Service serviceRef, string methodName, s
     foreach string path in resourcePath {
         resourceName += "$" + path;
     }
-    any resourceAnnotation = reflect:getResourceAnnotations(serviceRef, resourceName, RESOURCE_ANNOTATION,
-                                                            getModuleIdentifier());
+    string annotationQualifiedIdentifier = getModuleIdentifier() + ":ResourceConfig";
```

Comment:
Since this var is used inside the `getResourceAnnotation` function, shall we do it at java level. `getModuleIdentifier()` is also an interop call, I believe we can create this string from java level easily

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-http/pull/245/commits/e5fb411d9d70303024cd66cce3ed1f4b682b315f